### ISSUE TITLE
schedulemessage clear content?

### ISSUE THREAD

**Issue by HansElectrumpet** [AlexHarker/AHarker_Externals#18]
Hi Alex,

It is very nice to see the updates on the objects. I have just discovered schedulemessage and I was wondering if it is possible to clear future events from schedulemessage, like pipe has both the 'flush' method and just 'clear'. I could not find it in the code, so maybe this counts as a feature request?

Thanks.

Best, Hans.

**Comment by AlexHarker** [comment]
This doesn't currently exist - I'm not sure how difficult it would be to add - I think pipe might have it's own scheduler, which to my recollection schedulemessage does not currently, but I can certainly keep it here for consideration.

Out of interest - what is the sort of use case that you had in mind for this feature?

**Comment by HansElectrumpet** [comment]
Hi Alex,

Thanks for the quick reply. In my case the idea would be that I start a sequence of events but that I would also be able to cut off at any given moment. When then starting a new sequence again, I don't want any messages still in the pipeline disrupt this new sequence.

Those messages are OSC-messages with their parameters. I have found a workaround by converting the message and parameters to a symbol and then delay them with pipe and converting them back.

Best, Hans.

**Comment by AlexHarker** [comment]
Yes - that makes sense. No promises on when but this seems worth including if I can at some point.
